sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/NpmVersionHelper.java

```diff
@@ -44,6 +50,37 @@ public String getResolvedVersionFromNpmRegistry(PackageJson packageJson, String
 			throw new RuntimeException("Error retrieving resolved version from NPM", e);
 		}
 	}
+
+	public List<String> getPackageVersions(String packageName, int limit) {
+		try {
+			ProcessBuilder processBuilder = new ProcessBuilder()
+					.command("npm", "view", packageName, "versions", "--dry-run", "--json").redirectErrorStream(true);
+
+			Process process = processBuilder.start();
+
+			BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()));
```

Comment:
Wrap this in a try-with-resources so the reader always gets closed.

```suggestion
			try(BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()))) {
```

Also, instead of using the `BufferedReader` to read by line, please use the code from `getPackageMetadata()`; it's a little more efficient than a line reader for a case like this where we don't care about line-by-line format.